package-lock.json

```diff
@@ -9044,25 +9044,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Not sure why this change is here?

Reply:
To be honest, nor am I - this is the result of `nvm use` `npm install`. Now that I've finished up, I'll blast the entire node_modules directory and try again - that sometimes helps.

Reply:
Yeah, blowing up `node_modules` and these changes and reinstalling did nothing, so I've wiped this commit.